RestaurantReservation.Db.Test/RepositoriesTest/EmployeeRepositoryTest.cs

```diff
@@ -0,0 +1,120 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using RestaurantReservation.Db.Data;
+using RestaurantReservation.Db.Entities;
+using RestaurantReservation.Db.Enums;
+using RestaurantReservation.Db.Repositories;
+
+namespace RestaurantReservation.Db.Test.RepositoriesTest;
+public class EmployeeRepositoryTest : IDisposable
+{
+    private readonly DbContextOptions<RestaurantReservationDbContext> _options;
+    private readonly RestaurantReservationDbContext _context;
+    private readonly EmployeeRepository _repository;
+    private readonly IFixture _fixture;
+
+    public EmployeeRepositoryTest()
+    {
+        _options = new DbContextOptionsBuilder<RestaurantReservationDbContext>()
+            .UseInMemoryDatabase(databaseName: Guid.NewGuid().ToString())
+            .Options;
+
+        _context = new RestaurantReservationDbContext(_options);
+        _repository = new EmployeeRepository(_context);
+        _fixture = new Fixture();
```

Comment:
Maybe this is the tenth comment about the same thing, use `Fixture.Create`, it is easier, readable, and effecient.